README.md

```diff
@@ -157,16 +157,130 @@ The Docker image used in production, ci/Dockerfile, builds with the Python packa
-6. Click the __Run workflow__ button.
+To open a PR:
+
+- Make sure your branch is pushed up to the main repository
+
+- The title of the PR should be formated to contain the ticket number of the issue being worked with a title
+
+- The body should be filled out per the template that is provided
+
+- Add in the correct tags that represent the PR
+
+```markdown
+Title: #443 Issue Title
+
+
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes #443
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+## Checklist
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+```
+
+For tags, please make sure to add in additional tags to the PR based on the following categories:
+
+```yaml
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking-change
+    - title: Exciting New Features 🎉
+      labels:
+        - enhancement
+    - title: Bug Fixes 🐜
+      labels:
+        - bug
+        - hotfix
+    - title: Security Fixes 🔐
+      labels:
+        - cve
+        - security
+```
+
+If the PR requires additional manual validation/testing then make sure to add in the `test` tag to the PR as well.
+
+If the PR does not fit into these categories, then don't add a tag.
+
+## Release Process
+
+The portal releases are managed by tags in the GitHub repo.
+
+### Create a release for Perf
+
+To create a new release, the user will create a new version tag in the git repository. This can be for the current `HEAD` or for a specific commit from a PR. The tag format will be `v(Major).(Minor).(Hotfix)` format. So for version one of the portal we created `v1.0.0` and with a single hotfix a new tag was created for `v1.0.1`.
```

Comment:
```suggestion
To create a new release, the user will create a new version tag in the git repository. This can be for the current `HEAD` or for a specific commit from a PR. The tag format will be `v(Major).(Minor).(Hotfix)` format. The Portal uses a similar method, so for version one of the Portal we created `v1.0.0` and with a single hotfix a new tag was created for `v1.0.1`.
```